modules/images-configuration-registry-mirror-configuring.adoc

```diff
@@ -17,9 +17,7 @@ You can create postinstallation mirror configuration custom resources (CR) to re
 ifdef::winc[]
 [IMPORTANT]
 ====
-Windows images mirrored through `ImageDigestMirrorSet` and `ImageTagMirrorSet` objects have specific naming requirements. 
-
-include::snippets/wmco-mirror-naming-requirements.adoc[]
+Windows images mirrored through `ImageDigestMirrorSet` and `ImageTagMirrorSet` objects have specific naming requirements. For information, see "Using Windows containers with a mirror registry".
```

Comment:
Just wondering if there is an acceptable way to shorten this.
```suggestion
Windows images mirrored through `ImageDigestMirrorSet` and `ImageTagMirrorSet` objects have specific naming requirements, which you can find in "Using Windows containers with a mirror registry".
```

Reply:
> as described in "Using Windows containers with a mirror registry".  ??